policy/templates/subtemplates/auto/auto-test.gotmpl

```diff
@@ -198,6 +202,7 @@
 
       - name: Generate metadata and upload test reports
         id: metadata_report
+        if: always() && (steps.test_execution.conclusion != 'skipped') 
```

Comment:
Isn't this the same as always()? I think it should be just `steps.test_execution.conclusion != 'skipped'`

Reply:
this combination will make it run only when test execution has been executed, wether failed or not by using always, `steps.test_execution.conclusion != 'skipped'` will prevent metadata report from running if tests were not executed due to a previous failure like env up